### PR TITLE
feat: sheets component

### PR DESCRIPTION
**Pull request by alanneves**
In this PR I'm refactoring PbSheet to be equal to the prototype. It's not being used in simplificamais-app yet. So, it's not a breaking chang
dd
e

### CODE REVIEW

File: src/components/MenusAndToolbars/Sheets/Sheets.stories.mdx [hunk 1]
```diff
@@ -1,62 +1,136 @@
...
+      defaultValue: 'regular',
     },
   }}
 />
 
 export const Template = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
+  methods: {
+    onSelectSheet(value) {
+      console.log('selected sheet', value);
```
Comment (mickaelnn) [inline src/components/MenusAndToolbars/Sheets/Sheets.stories.mdx:50]: debug?

**Review by mickaelnn — CHANGES_REQUESTED** [review]

File: src/components/MenusAndToolbars/Sheets/Sheets.stories.mdx (hunk 1 above)
Comment (alanneves) [reply src/components/MenusAndToolbars/Sheets/Sheets.stories.mdx:50]: no, this is to prove that the event works.

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by EvertonLMsilva — APPROVED** [review]
LGTM

**Pull request merged**